ktbridge/build.gradle.kts

```diff
@@ -4,13 +4,17 @@ plugins {
   id("org.jetbrains.dokka")
 }
 
+val copyTestingJs = tasks.register<Copy>("copyTestingJs") {
+  destinationDir = buildDir.resolve("generated/testingJs")
+  from("testing/build/distributions/testing.js")
+  dependsOn(":ktbridge:testing:jsBrowserProductionWebpack")
```

Comment:
Still not perfect since we're depending on implementation detail paths, task names, and doing both across project boundaries, but we can "fix" (i.e., adhere to Gradle's ridiculous ways of doing things "correctly") later.